section3/exercises/arrays.rb

```diff
@@ -15,55 +15,65 @@
 # EXAMPLE: write code below that will print an array of animals.
 # Store the array in a variable.
 animals = ["Zebra", "Giraffe", "Elephant"];
-print animals
+puts animals
 
 # EXAMPLE: Write code below that will print "Zebra" from the animals array
 # YOUR CODE HERE
-print animals[0]
+puts animals[0]
 
 # YOU DO: Write code below that will print the number of elements in array of
 # animals from above.
-
+puts animals.length
 
 # YOU DO: Write code that will reassign the last item in the animals
 # array to "Gorilla"
-
+animals[2] = "Gorilla"
+puts animals
 
 # YOU DO: Write code that will add a new animal (type of your choice) to position 3.
-
-
+animals[3] = "Lemur"
+puts animals
 # YOU DO: Write code that will print the String "Elephant" in the animals array
-
+animals[4] = "Elephant"
+puts animals[4]
 
 #-------------------
 # PART 2: Foods: Array Methods
 #-------------------
+puts '-' * 10
 
 # YOU DO: Declare a variable that will store an an array of at least 4 foods (strings)
-
+refridgerator = ["Fruit", "Chicken", "Eggs", "Milk"];
 
 # YOU DO: Write code below that will print the number of elements in the array of
 # foods from above.
-
+puts refridgerator.length
 
 # YOU DO: Write code below that uses a method to add "broccoli" to the foods array and
 # print the changed array to verify "broccoli" has been added
+puts '-' * 10
 
+refridgerator[4] = "broccoli"
```

Comment:
Challenge yourself: is there another way to add a new element to the end of an array in Ruby?